app/assets/javascripts/admin/slides.js

```diff
@@ -172,6 +173,10 @@ Utils.fireWhenReady(['slides#index'], function(e) {
   $('#filters a, #sort a').on('click', AdminSlides.filterClicked);
 });
 
-Utils.fireWhenReady(['slides#show'], function() {
+Utils.fireWhenReady(['slides#show'], function(e) {
+  // TODO: Remove when materialize releases autocomplete update or another fix
+  AddOwnerMonkeyPatch.setURL("/slides/autocomplete_user_email?term=");
```

Comment:
One concern with this method: I think AddOwnerMonkeyPatch functions as a singleton. So calling it twice means that the value will end up being the same for both instances.

I don't expect this causes problems because both endpoints do the same thing.

If that is the case, you could just make it a hardcoded property of the module (maybe with a slightly more descriptive name) and eliminate the `setURL` method:

```
var AddOwnerMonkeyPatch = (function() {

  var autocomplete_endpoint = "/slides/autocomplete_user_email?term=";
```